schema-engine/sql-introspection-tests/tests/postgres/mod.rs

```diff
@@ -224,3 +224,116 @@ async fn index_sort_order_stopgap(api: &mut TestApi) -> TestResult {
 
     Ok(())
 }
+
+#[test_connector(tags(Postgres), exclude(CockroachDb))]
+async fn check_constraints_stopgap(api: &mut TestApi) -> TestResult {
+    // https://www.notion.so/prismaio/Indexes-Constraints-Check-constraints-PostgreSQL-cde0bee25f6343d8bbd0f7e84932e808
+
+    let schema = indoc! {r#"
+      CREATE TABLE products (
+          product_id serial PRIMARY KEY,
+          name text,
+          price numeric CHECK (price > 0)
+      );
+    "#};
+
+    api.raw_cmd(schema).await;
+
+    let expectation = expect![[r#"
+        generator client {
+          provider = "prisma-client-js"
+        }
+
+        datasource db {
+          provider = "postgresql"
+          url      = "env(TEST_DATABASE_URL)"
+        }
+
+        model products {
+          product_id Int      @id @default(autoincrement())
+          name       String?
+          price      Decimal? @db.Decimal
+        }
+    "#]];
+
+    api.expect_datamodel(&expectation).await;
+
+    let expectation = expect![[r#"
+        [
+          {
+            "code": 31,
+            "message": "These tables have check constraints, which are not yet fully supported. Read more: https://pris.ly/d/postgres-check-constraints",
+            "affected": [
+              {
+                "name": "products_price_check",
+                "definition": "CHECK ((price > (0)::numeric))"
```

Comment:
Nice, that will be interesting to see how it deals with all the ones we should have in Introspection-CI 👍